widget/RazorpayElementsSubscriptionButton.php

```diff
@@ -168,7 +168,15 @@ protected function render() {
 			?>
 			
 			<form>
-				<script src="https://cdn.razorpay.com/static/widget/subscription-button.js" data-plugin="<?php esc_attr_e($dataPlugin) ?>" data-subscription_button_id="<?php esc_attr_e( ! empty( $settings['select_button'] ) ? $settings['select_button'] : '' ); ?>"> </script>
+			<?php
+			wp_print_script_tag(
```

Comment:
@neera11 can you please test this tag with below WordPress 5.7.0 version once. If it's only supporting 5.7.0 onwards then we need to mention it in the doc also.

Reply:
@neera11 Looks like this `wp_print_script_tag ` wont work below 5.7 versions. Can you please mention it in the readme file. Also need to convey the same to the WordPress team so that if any alternative is there, we can apply here.